WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -532,6 +573,34 @@ class PageListViewController: AbstractPostListViewController, UIViewControllerRe
         }
     }
 
+    private func addSetParentAction(to controller: UIAlertController, for page: AbstractPost, at index: IndexPath?) {
+        /// This button is disabled for self-hosted sites
+        //
+        guard blog.supports(.wpComRESTAPI) else {
```

Comment:
This is a feature that core has too, right? Is the API different enough that it makes it difficult to implement the XML-RPC version, or is it just the case of handling WPCom API first?

Reply:
The second you said at the moment. I got and I still have problem with XMLRPC version. We actually read and store the value from the get posts but we don't update it. Also if you try to update that value the POST response of the UPDATE action is success but I still get the previous values from the get posts when the app sync again the pages list. I'm still investigating why.